python/requirements.txt

```diff
@@ -13,4 +13,10 @@ nest_asyncio
 prompt_toolkit
 mock
 pytest-asyncio
-lastmile_utils==0.0.3
\ No newline at end of file
+lastmile_utils==0.0.3
+transformers
```

Comment:
lets pin transformers. I noticed their api docs state that the api will change over time, don't want this to fail unexpectedly.